src/main/java/com/mathworks/ci/MatlabBuilder.java

```diff
@@ -306,7 +378,22 @@ public void setTaJunitChkBx(boolean taJunitChkBx) {
         public void setTaCoberturaChkBx(boolean taCoberturaChkBx) {
             this.taCoberturaChkBx = taCoberturaChkBx;
         }
-
+        
+        @DataBoundSetter
+        public void setTaPDFReportChkBx(boolean taPDFReportChkBx) {
```

Comment:
Is this public primarily for testability?

Reply:
No, the setter method needs to be public because it is being accessed by the internal classes of Jenkins (Stapler classes) whenever user clicks the check boxes in the UI.